### PR TITLE
Manually pushing the first java client

### PR DESCRIPTION
**Pull request by rmadisonhaynie**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+                <executions>
+                    <execution>
+                        <id>add_sources</id>
+                        <phase>generate-sources</phase>
+                        <goals>
+                            <goal>add-source</goal>
+                        </goals>
+                        <configuration>
+                            <sources>
+                                <source>src\main\java</source>
```
Comment (yuancheng-p) [inline pom.xml]: When generated on MacOS, it's `src/main/java` here.
I would prefer to keep `src/main/java`.

File: build.gradle [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+install {
+    repositories.mavenInstaller {
+        pom.artifactId = 'marketing.java-client'
+    }
+}
+
+task execute(type:JavaExec) {
+   main = System.getProperty('mainClass')
+   classpath = sourceSets.main.runtimeClasspath
+}
```
Comment (yuancheng-p) [inline build.gradle]: was there any change of the task `execute` in the sdk-generator ? It seems the master version of sdk-generator doesn't generate the `execute` task.

File: README.md [hunk 3]
```diff
@@ -1,10 +1,189 @@
-# criteo-java-marketing-sdk
+# Criteo Marketing SDK for Java
```
Comment (yuancheng-p) [inline README.md:1]: This README is nice, it would be better to merge https://github.com/criteo/criteo-marketing-sdk-generator/pull/13 first ;)

**Review by yuancheng-p — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (rmadisonhaynie) [reply pom.xml]: good catch

File: build.gradle (hunk 2 above)
Comment (rmadisonhaynie) [reply build.gradle]: Yes this should have been removed

**Review by yuancheng-p — COMMENTED** [review]
https://github.com/criteo/criteo-marketing-sdk-generator/pull/14 will automatically generate the first version, so we don't need to do the manual push \o/.
Thanks to this PR, we have discovered a lot of subtle changes in the generated clients.
Let's keep it as automatic as possible.

**Pull request closed without merging**